### PR TITLE
Feat: Added Category and Rules page

### PR DESCRIPTION
**Pull request by vinc3nati**
- categories page
- rules page

image

image

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]
Clean code and UI
Looks good to merge

File: src/components/CategoryCard/CategoryCard.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Link, useLocation } from "react-router-dom";
+import { QuizModelType } from "../../types/quiz.types";
+
+export const CategoryCard = ({ data }: { data: QuizModelType }) => {
+  console.log("Card :", data);
```
Comment (manish591) [inline src/components/CategoryCard/CategoryCard.tsx]: Remove console

File: src/pages/Category/Category.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const categories = QuizModel.reduce(
+    (acc, curr) =>
+      acc.includes(curr.quizCategory) ? acc : [...acc, curr.quizCategory],
+    [] as string[]
+  );
+
+  //   useEffect(() => {
+  //     dispatch({ type: "RESET" });
+  //   }, []);
+
```
Comment (manish591) [inline src/pages/Category/Category.tsx:28]: Remove commented code

**Review by manish591 — APPROVED** [review]
The code is looking good. 
Properly indented.

**Pull request merged**